cob_moveit_bringup/launch/stomp_moveit_planning_pipeline.launch.xml

```diff
@@ -0,0 +1,20 @@
+<launch>
```

Comment:
remove this file...the required `stomp_planning_pipeline.xml` is part of this PR below